app/Services/StatsServiceProvider.js

```diff
@@ -139,6 +139,52 @@ async function getNumberOfUnactivatedVocabulary({ languagePackageId, groupId, us
   return number;
 }
 
+// get number of today learned vocabs
+async function getNumberOfLearnedTodayVocabulary({ languagePackageId = null, userId }) {
+  const languagePackage = await LanguagePackage.findOne({
+    attributes: languagePackageId
+      ? ['vocabsPerDay']
+      : [[sequelize.fn('sum', sequelize.col('vocabsPerDay')), 'vocabsPerDay']],
+    where: {
+      ...(languagePackageId ? { id: languagePackageId } : {}),
+      userId,
+    },
+  });
+
+  const number = await PackageProgress.findOne({
+    attributes: languagePackageId
+      ? [
+          ['learnedTodayCorrect', 'correct'],
```

Comment:
PackageProgress does not have a ``correct`` attribute

Reply:
the nested array only renamed them, so it is available.